app/src/main/java/com/sun_asterisk/moviedb_50/data/model/Movie.kt

```diff
@@ -0,0 +1,28 @@
+package com.sun_asterisk.moviedb_50.data.model
+
+data class Movie(
+        var movieID: Int?,
+        var movieTitle: String?,
+        var movieOverview: String?,
+        var moviePosterPath: String?,
+        var movieBackdropPath: String?,
+        var movieVoteCount: Int?,
+        var movieVoteAverage: Double?,
+        var moviePopularity: Double?,
+        var movieReleaseDate: String?,
+        val movieGenreID: IntArray?
+) {
+  object MovieEntry {
+    const val MOVIE = "results"
```

Comment:
Bạn xem lại format code nhé
```suggestion
    object MovieEntry {
        const val MOVIE = "results"
```